### ISSUE TITLE
gpt3.5部署成功,gpt4就错误

### ISSUE THREAD

**Issue by zyjia** [haibbo/cf-openai-azure-proxy#68]
前端都是使用https://github.com/Yidadaa/ChatGPT-Next-Web不是,3.5的没有问题,gpt4总数返回{
  "error": {
    "code": "404",
    "message": "Resource not found"
  }
}
也不知道是这个项目还是ChatGPT-Next-Web的问题

**Comment by alect001** [comment]
我没有这个问题，我所有模型都成功。

**Comment by crypticatul** [comment]
Yes, I am too having this problem in Chatgpt next web

Failed to load resource: the server responded with a status of 404 ()

**Comment by haibbo** [comment]
Is there a problem with using other tools to call GPT-4? Such as a chatbox
Can you provide some screenshots? @crypticatul @zyjia

**Comment by crypticatul** [comment]
> Is there a problem with using other tools to call GPT-4?  @crypticatul @zyjia

No, I have just tried on ChatGPT NExt Web. Here is screenshots.
<img width="562" alt="image" src="https://github.com/haibbo/cf-openai-azure-proxy/assets/88734298/7586168b-7b11-44b7-8f2e-168bdd7a78c1">

<img width="859" alt="image" src="https://github.com/haibbo/cf-openai-azure-proxy/assets/88734298/25e0af00-338e-4bd7-8237-256976b129b2">

**Comment by psywxn** [comment]
@zyjia @haibbo 
Hello!
我在使用zoterogpt时遇到了同样的问题。

我在创建Azure OpenAI服务时，已经不能选择美东地区（这样无法部署GPT4，我查看了Azure的帮助文档，似乎这个地区已经满了）。我实际选择了Canada East，并分别部署了gpt3.5和gpt4两种模型。

<img width="573" alt="1705648172431" src="https://github.com/haibbo/cf-openai-azure-proxy/assets/69781730/cf0b9024-25fc-4417-ac9d-a8a8d79a5942">
此时报错和lz一样，我换成gpt3.5就成功了。我想到会不会和地区有关，于是把代理节点换成了加拿大，之后就成功了。

Hope it helps!

**Comment by ShinChven** [comment]
可以写个Map来做多节点。根据不同的模型，访问不同的 Azure 节点。

**Comment by phakcc** [comment]
> @zyjia @haibbo Hello! 我在使用zoterogpt时遇到了同样的问题。
> 
> 我在创建Azure OpenAI服务时，已经不能选择美东地区（这样无法部署GPT4，我查看了Azure的帮助文档，似乎这个地区已经满了）。我实际选择了Canada East，并分别部署了gpt3.5和gpt4两种模型。
> 
> <img alt="1705648172431" width="573" src="https://private-user-images.githubusercontent.com/69781730/297982342-cf0b9024-25fc-4417-ac9d-a8a8d79a5942.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcwMzUwMjcsIm5iZiI6MTcwNzAzNDcyNywicGF0aCI6Ii82OTc4MTczMC8yOTc5ODIzNDItY2YwYjkwMjQtMjVmYy00NDE3LWFjOWQtYThhOGQ3OWE1OTQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA0VDA4MTg0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM5NTdhZjNlZTY5ZGQ2NzdjM2U2YWE2OGY4NWRiNjZmOTFjMDZjOGM2MGU5YjY2ZDk1YWFmMTIzMjQzMmM2ZDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qh1cm-TCdGBarqTJmiFjkeZkkoukf_CKLwCgS0Lqr7I"> 此时报错和lz一样，我换成gpt3.5就成功了。我想到会不会和地区有关，于是把代理节点换成了加拿大，之后就成功了。
> Hope it helps!

瑞典中部也可以，这个区域还有 Dall E 3
